e2e/timeline.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect, Page } from "@playwright/test";
+
+test("Public timeline shows all messages", async ({ page }) => {
+  await page.goto("/public");
+  await expect(
+    page.getByText("This is a message written by user 2")
+  ).toBeVisible();
+  await expect(
+    page.getByText("This is a message written by user 3")
+  ).toBeVisible();
+});
```

Comment:
This seems to be idempotent in my eyes:
If we generate a big enough test suite, we would have tests generating posts flooding this front page and maybe the posts will no longer be available. As these two messages stems from the data seeding, we expect at least 2 tweets to always be visible. Could we have a `.getBy` that matches all messages on the public timeline and asserts that at least 2 tweets exists instead? Then we could have a separate test that verifies the text is properly displayed (i.e. creating a new post will show up on public timeline and match on the exact test?)

Reply:
You're welcome to do so, however it seems like a non-issue to me. In my mind we would never be creating more than 30 tests (The point where this becomes an issue) with messages being posted as the only thing that tests, is that you are able to post messages.

Reply:
Sure, let's address this if we get near the 30 post limit for testing.